src/spatial/models/heatmapModel.py

```
@@ -0,0 +1,799 @@
+                client = redis.Redis(
+                    host=redis_host,
+                    port=int(redis_port),
+                    db=int(os.getenv("REDIS_DB", "0")),
+                    password=os.getenv("REDIS_PASSWORD"),
+                    decode_responses=False,
+                )
+                client.ping()
+                self.logger.info("Redis connection successful via host/port.")
+                return client
+
+        except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)